SofaKernel/framework/sofa/simulation/MechanicalVisitor.h

```diff
@@ -1333,15 +1372,12 @@ class SOFA_SIMULATION_CORE_API MechanicalPropagatePositionVisitor : public Mecha
     SReal t;
     sofa::core::MultiVecCoordId x;
     bool ignoreMask;
-    bool applyProjections;
```

Comment:
ok to remove all the `applyProjections` from all the propagate visitors.
But a doc should clearly state to run the new visitor `MechanicalProjectPositionAndVelocityVisitor` before propagation.

I guess there are cases where we want to project only one vector (coord or deriv), but it would be easy to write other visitors for that.